docs/README-DEPLOYMENT.md

```diff
@@ -14,53 +14,75 @@ Before generating the documentation, set [Docker's runtime memory limit](https:/
 
 The recommended runtime memory limit is 8 GB. It allows us to generate 4 documentation versions at a time.
 
-## Deploying the documentation from the command line
+## Deploying the documentation
 
-To deploy the documentation from the command line:
+Handsontable's [GitHub Actions setup](https://github.com/handsontable/handsontable/actions) deploys the documentation based on the following Docker image tags:
+
+| Docker image tag      | Build type | Triggered by                                          | Used for                                                                                                                                   |
+| --------------------- | ---------- | ----------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| `:[COMMIT_HASH]`      | Staging    | A push that changes `docs/**` on any branch           | [Manual local deployment](#deploying-the-documentation-locally-at-a-specific-commit)                                                       |
+| `:latest`             | Staging    | A push that changes `docs/**` on the `develop` branch | Automatic staging deployment<br><br>[Manual staging deployment](#manually-deploying-the-documentation-to-the-staging-environment)          |
+| `:production`         | Production | A push that changes `docs/**` on the `develop` branch | Automatic production deployment<br><br>[Manual production deployment](#manually-deploying-the-documentation-to-the-production-environment) |
+| `:prod-[COMMIT_HASH]` | Production | A push that changes `docs/**` on the `develop` branch | Automatic production deployment backup                                                                                                     |
+
+### Deploying the documentation locally at a specific commit
+
+To deploy the documentation locally, at a specific commit:
+
+```bash
+npm run docs:review [COMMIT_HASH]
+```
```

Comment:
I think that this section should be moved to `README-EDITING.md`, because it is not really deployment.

Also, please add information that Docker Desktop is required and where to get it from.